### ISSUE TITLE
Higurashi changes/4:3 monitor and sprites

### ISSUE THREAD

**Issue by randoomdude** [07th-mod/higurashi-console-arcs#593]
I wanted to play the game on my second 4:3 monitor occasionally, but even if I change the resolution and set it to old 4:3 backgrounds, the game is stuck in 16:9/can't fullscreen on 4:3 without it sticking out

Wanted to have mouth flaps for the mangagamer sprites

Wanted to be able to mix and match different sprites, like have a mangagamer sprite just for Ooishii, and have the rest be console sprites, or any combination

Not sure which of these would be easisest to implement, but any changes would be nice if possible

**Comment by randoomdude** [comment]
Also a way to have auto play ignore the voice thing where it slows to play the line, so voicelesss auto would be nice to implement

**Comment by drojf** [comment]
> I wanted to play the game on my second 4:3 monitor occasionally, but even if I change the resolution and set it to old 4:3 backgrounds, the game is stuck in 16:9/can't fullscreen on 4:3 without it sticking out

I did this in the past because it was difficult trying to get everything to work properly when our main mod is in 16:9. Most things would work, but occasionally something might not work properly.

However, I could add an "Experimental 4:3" mode, without trying to fix any bugs that happen in this mode (unless they're easy to fix), so people don't expect too much.

If you modify the line `SetScreenAspect("0.5625");` to `SetScreenAspect("0.75");` in the `HigurashiEp01_Data\StreamingAssets\Update\init.txt` file, you can see how this would look.

> Wanted to have mouth flaps for the mangagamer sprites

Not possible as no such mangagamer sprites exist with different mouths. Only console sprites (taken from the console game which had lipsync) have different mouths.

> Wanted to be able to mix and match different sprites, like have a mangagamer sprite just for Ooishii, and have the rest be console sprites, or any combination

Possible, but I think you're the first person to ever ask for this feature. I don't think enough people would want this feature so that it'd be worth adding it.

Theoretically you could copy around some game files to make this work (copy Oooishii's mangagamer sprites into the console folder), but you would lose lipsync for that character (see last question).

> Also a way to have auto play ignore the voice thing where it slows to play the line, so voicelesss auto would be nice to implement

This is probably easy enough to add, but I haven't messed with auto-mode before. I remember I needed to fix auto-mode for Rei so I might get to it eventually. (Note to self: simply skipping voice file playback if the voice volume is zero or this mode is enabled might be enough to fix this?)

----

In summary, I could pretty easily add an experimental 4:3 mode, and also auto-mode which ignores voices in its timing, but the rest would either be impossible or not worth adding vs how many users requested it.

Do you still have quite a bit left of the console arcs/Higurashi to play? Then you could test these two features for me.

**Comment by randoomdude** [comment]
yeah even if it's experimental that would be nice? if have quite a bit left, on chapter 3

**Comment by randoomdude** [comment]
I don't know if you have the "experimental mode" set up yet, but I tried to look in the init.txt file just to make it 4:3, and I can't find it in the file to change
![image](https://user-images.githubusercontent.com/6027132/202308821-ceacbcb6-7796-4fe0-93ea-b3035d55c263.png)

**Comment by drojf** [comment]
ah, I got the experimental 4:3 aspect working, but since I hadn't done the voice feature I didn't post anything here yet.

Overwrite your existing Assembly-CSharp.dll in the folder `Higurashi 03 - Tatarigoroshi\HigurashiEp03_Data\Managed\Assembly-CSharp.dll`

[Assembly-CSharp-tatari-ch3-exp-43.zip](https://github.com/07th-mod/higurashi-console-arcs/files/10026225/Assembly-CSharp-tatari-ch3-exp-43.zip)

At the bottom of the graphics menu there should be an option for 4:3 (you need to choose old backgrounds and disable background stretching to enable it, otherwise it will have no effect)

**Comment by drojf** [comment]
OK, I can already see a few problems, noting down for myself later:
 - If you are in 4:3 aspect, then you change resolutions, it resets the aspect to 16:9
 - If the resolution is 720p or less, and you use 4:3 mode, then the mod menu is cut off. 1080p and above works fine
 
 I feel stupid but I never actually tested fullscreen with 4:3 mode, which...doesn't work. Gimme some time and I'll try to fix it lol

**Comment by randoomdude** [comment]
thanks man, appreciate the fix
